app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+
+    module ActiveRecord
+    class Base
+      def self.random
+        if (c = count) != 0
+          find(:first, :offset =>rand(c))
+        end
+      end
+    end
```

Comment:
🤔 It's usually not recommended to modify `ActiveRecord::Base` directly and if you need a method to be inherited to all models, you can define it in `ApplicationRecord. In the rare case where that is true, the method would be defined in `app/models/application_record.rb` not in the ApplicationController class.

Controllers should not be responsible for model logic.

Only if `random` is a method that you want defined for *every*, Model, should you define it in `ApplicationRecord` otherwise I would keep it in the `Meal` class.

What do you think?